### ISSUE TITLE
Replacing 'routerReducer' as an injectable token rather than static text

### ISSUE THREAD

**Issue by phillipzada** [ngrx/platform#410]
## I'm submitting a...

<pre><code>
[ ] Regression (a behavior that used to work and stopped working in a new release)
[ ] Bug report  
[x] Feature request
[ ] Documentation issue or request
</code></pre>

## What is the current behavior?
Currently in router_store_module.ts routerReducer is a static text key. 

## Expected behavior:
Allow for the key name to be injected via a provider and token but defaulting to 'routerReducer' of not provided

## Other information:
Happy to put together a PR if approved.

**Comment by brandonroberts** [comment]
Seems reasonable to me. I would suggest adding a `forRoot` method that takes a config object with a key for the router reducer name. This way you don't have to register a new provider to override it.

**Comment by PhillipZada** [comment]
Cool, so the end result will be something like this

```
StoreRouterConnectingModule.forRoot( { stateName: 'routerReducer' } );
```

**Comment by brandonroberts** [comment]
Yep. What about `stateKey` for the name?

**Comment by edmundo096** [comment]
This catched me by surprise. I used another name for the router state key and time traveling didn't worked on my first try.
After checking the example-app, I noticed the persistence on using the key 'routerReducer', so I checked the src and found that key. I gave it a try and it worked.

At least the setup code on the docs makes mention it in the `StoreModule.forRoot({ routerReducer: routerReducer }),` line, but maybe a note would be useful either if this is changed or not.

**Comment by pauldubois777** [comment]
@edmundo096 - Thanks for your post!  This was driving me crazy trying to debug why time-traveling wasn't working!  I tried a lot of things, and did a lot of searching for fixes until I found your post.  I renamed the property to `routerReducer` and now it works!

I hope this can be fixed so that specific naming of the property isn't required, or at least update the documentation.  I can help if needed.
